### ISSUE TITLE
Brower console is logging "Some cookies are misusing the recommended "sameSite" attribute

### ISSUE THREAD

**Issue by epugh** [o19s/quepid#130]
**Describe the bug**
In the browser console on Firefox:

```
Cookie “_quepid_session” will be soon rejected because it has the “sameSite” attribute set to “none” or an invalid value, without the “secure” attribute. To know more about the “sameSite“ attribute, read https://developer.mozilla.org/docs/Web/HTTP/Cookies
```

**To Reproduce**
Bring up quepid in modern brower, like Firefox 76, and check console.

**Expected behavior**
no warning message

**Comment by epugh** [comment]
Here is where it was fixed in newer versions of Rails: https://github.com/rails/rails/pull/28297

**Comment by epugh** [comment]
Fixed in #131
